FrameworkSystem/private/standardLogging/LoggingRoot.py

```diff
@@ -210,3 +213,23 @@ def __configureLevel(self):
       self._setLevel(LogLevels.DEBUG)
       self.showHeaders(True)
       self.showThreadIDs(True)
+
+  def enableLogsFromExternalLibs(self, isEnabled=True):
```

Comment:
This function signature looks confusing. I would suggest to make two functions:

  enableLogsFromExternalLibs()
  disableLogsFromExternalLibs()

Those can just call the original function with the corresponding flag